### DISCUSSION TITLE
Can't install add-on

### DISCUSSION THREAD

**Discussion by detlefs** [FunkeyFlo/ps5-mqtt#252] [Q&A] [answered]
Maybe it's a stupid mistake, but I'm stuck trying to install the PS5 MQTT add-on...
- I installed Mosquitto broker and it's running
- I added the repository to the add-in store (via the button on your readme.md page), and clicked the Update function from the menu

Now, I don't find a PS5 MQTT add-on. When I search for "PS5" nothing is found...

What am I missing?

**Comment by FunkeyFlo** [comment, marked as answer]
I had the same issue this week with another add on. I resolved it by (on Android) closing the HA companion app and re-opening it. On a browser I'd advise clearing your cache.

**Reply by detlefs** [reply]
Thanks. Clearing cache was the trick
